src/inmanta/module.py

```diff
@@ -1098,25 +1098,30 @@ def get_all_submodules(self) -> List[str]:
 
         return modules
 
-    def load_plugins(self) -> None:
+    def get_plugin_files(self) -> Iterator[Tuple[str, str]]:
```

Comment:
Not sure if this will break typing or improve. But you can redefine strings to keep things a like a path and a module name seperate. For example: https://github.com/inmanta/inmanta/blob/master/src/inmanta/data/model.py#L102

@wouterdb @sanderr wdyt?

Reply:
I think it would be an improvement.